jsapp/js/account/stripe.api.ts

```diff
@@ -48,13 +50,62 @@ export async function changeSubscription(
   });
 }
 
-export const useOrganizationQuery = () => useQuery({
-  queryFn: async () => {
-    const response = await fetchGet<PaginatedResponse<Organization>>(endpoints.ORGANIZATION_URL);
-    return response.results?.[0];
-  },
-  queryKey: [QueryKeys.organization],
-});
+export const useOrganizationQuery = () => {
+  const isMmoEnabled = useFeatureFlag(FeatureFlag.mmosEnabled);
+
+  // Using a separated function to fetch the organization data to prevent
+  // feature flag dependencies from being added to the hook
+  const fetchOrganization = async (): Promise<Organization> => {
+    // Once we have the full data from backend, this function should be modified to return
+    // the actual data.
+
+    if (isMmoEnabled) {
+
+      const currentAccount = sessionStore.currentAccount as AccountResponse;
+
+      if (!currentAccount) {throw new Error('No account data found');}
```

Comment:
React query question, would this be sufficient for, later in some component, for the check `orgQuery.isError` to return true?

Reply:
Exactly. React query catches any error and sets its error flag automatically.